Source/RapyutaSimulationPlugins/Private/Core/RRPlayerController.cpp

```diff
@@ -42,26 +35,31 @@ void ARRPlayerController::BeginPlay()
 
 bool ARRPlayerController::Initialize()
 {
-    if (false == IsNetMode(NM_Standalone))
+    if (false == IsNetMode(NM_Standalone) ||
+        nullptr == GameMode ||
+        !GameMode->IsDataSynthSimType()
+        )
     {
         return true;
     }
+    
+#if RAPYUTA_USE_SCENE_DIRECTOR
     verify(GameState->HasInitialized(true));
     verify(GameState->HasSceneInstance(SceneInstanceId));
+#endif
 
     ActorCommon = URRActorCommon::GetActorCommon(SceneInstanceId);
+
+#if RAPYUTA_USE_SCENE_DIRECTOR
     check(ActorCommon);
     check(ActorCommon->SceneInstanceId == SceneInstanceId);
 
     SceneCamera = ActorCommon->SceneCamera;
     check(SceneCamera);
-    if (GameMode->IsDataSynthSimType())
-    {
-        Possess(SceneCamera);
-    }
```

Comment:
I suppose we should keep this since might be presumptuous to automatically `Possess(SceneCamera)`, which will lock user camera view,